packages/context/test/unit/context.unit.ts

```diff
@@ -272,6 +272,24 @@ describe('Context', () => {
     });
   });
 
+  describe('findByTagValues', () => {
+    it('returns matching binding', () => {
+      const b1 = ctx.bind('foo').tag({name: 'my-controller'});
+      ctx.bind('bar').tag('t2');
+      ctx.bind('baz').tag({weight: 1});
```

Comment:
Reposting https://github.com/strongloop/loopback-next/pull/1262#discussion_r182082629:

Would you mind using a more descriptive strings than bar/baz/t1/t2?

For example:
 - foo => ProductController
 - bar => ProductRepository
 - baz => db
 - t2 => repository
 - t3 => datasource